cmd/rollappd/start.go

```diff
@@ -118,14 +119,18 @@ is performed. Note, when enabled, gRPC will also be automatically enabled.
 		},
 		RunE: func(cmd *cobra.Command, _ []string) error {
 			serverCtx := server.GetServerContextFromCmd(cmd)
-
 			log_path := serverCtx.Viper.GetString(flagLogFile)
 			fileInfo, err := os.Stat(log_path)
 			if err == nil && !fileInfo.IsDir() {
 				serverCtx.Logger.Info("Using rotating file for logging", "log_path", log_path)
+				loggerMaxSize, err := strconv.Atoi(os.Getenv("MAX_LOG_SIZE"))
```

Comment:
@omritoptix use Viper, not os.Getenv directly
It will have default value built in